bin/pluto.jl

```diff
@@ -44,6 +46,14 @@ function convert_notebook(path)
         """
         ---
         title: "$(title)"
+        format:
+        html:
+            embed-resources: true
+            self-contained-math: true
```

Comment:
Was this indentation intentional? I thought `html` was a subkey of `format`.

Reply:
Good catch. Fixed